### ISSUE TITLE
edit Content page spits error: Undefined property: Zikula\PageLockModule\Api\LockingApi::$entityManager

### ISSUE THREAD

**Issue by MrMontesa** [zikula/core#3093]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.4.3
| PHP Version       | 5.6.25

After fixing #3091 new error about entityManager appears
### Expected behavior
Properly edit page

### Actual behavior
receiving error 500 with message:

`Notice: Undefined property: Zikula\PageLockModule\Api\LockingApi::$entityManager`

Stack Trace:

```
in system/PageLockModule/Api/LockingApi.php at line 216   -
            $newLock->setTitle($lockedByTitle);
            $newLock->setIpno($lockedByIPNo);
            // write this back to the db
            $this->entityManager->persist($newLock);
        }
        $this->entityManager->flush();
at ErrorHandler ->handleError ('8', 'Undefined property: Zikula\PageLockModule\Api\LockingApi::$entityManager', '/data/web/1/000/027/915/75642/htdocs/staging.amc-idstein/system/PageLockModule/Api/LockingApi.php', '216', array('lockName' => 'contentPage2', 'lockedByTitle' => 's.waitz', 'lockedByIPNo' => '62.96.145.145', 'sessionId' => '', 'theSessionId' => 'k460pi703j5c5km87ahihu7p46', 'locks' => array(), 'count' => '0', 'expireDate' => object(DateTime), 'newLock' => object(PageLockEntity))) 
in system/PageLockModule/Api/LockingApi.php at line 216   + 
at LockingApi ->requireLock ('contentPage2', 's.waitz', '62.96.145.145') 
in system/PageLockModule/Api/LockingApi.php at line 143   + 
at LockingApi ->addLock ('contentPage2', 'http://staging.amc-idstein.de/index.php?module=content&type=admin&func=main', false) 
in system/PageLockModule/Api/UserApi.php at line 44   + 
at UserApi ->pageLock (array('lockName' => 'contentPage2', 'returnUrl' => 'http://staging.amc-idstein.de/index.php?module=content&type=admin&func=main')) 
at call_user_func (array(object(UserApi), 'pageLock'), array('lockName' => 'contentPage2', 'returnUrl' => 'http://staging.amc-idstein.de/index.php?module=content&type=admin&func=main')) 
in lib/util/ModUtil.php at line 1222   + 
at ModUtil ::exec ('PageLock', 'user', 'pageLock', array('lockName' => 'contentPage2', 'returnUrl' => 'http://staging.amc-idstein.de/index.php?module=content&type=admin&func=main'), true, null) 
in lib/util/ModUtil.php at line 1298   + 
at ModUtil ::apiFunc ('PageLock', 'user', 'pageLock', array('lockName' => 'contentPage2', 'returnUrl' => 'http://staging.amc-idstein.de/index.php?module=content&type=admin&func=main')) 
in modules/Content/lib/Content/Form/Handler/Admin/EditPage.php at line 78   + 
at Content_Form_Handler_Admin_EditPage ->initialize (object(Zikula_Form_View)) 
in lib/legacy/Zikula/Form/View.php at line 283   + 
at Zikula_Form_View ->execute ('admin/editpage.tpl', object(Content_Form_Handler_Admin_EditPage)) 
in modules/Content/lib/Content/Controller/Admin.php at line 54   + 
at Content_Controller_Admin ->editpage (array('pid' => '2')) 
at call_user_func_array (array(object(Content_Controller_Admin), 'editPage'), array(array('pid' => '2'))) 
in lib/legacy/Zikula/AbstractController.php at line 153   + 
at Zikula_AbstractController ->__call ('editPageAction', array(array('pid' => '2'))) 
at Content_Controller_Admin ->editPageAction (array('pid' => '2')) 
at call_user_func (array(object(Content_Controller_Admin), 'editPageAction'), array('pid' => '2')) 
in lib/util/ModUtil.php at line 1222   + 
at ModUtil ::exec ('Content', 'admin', 'editPage', array('pid' => '2'), false, null) 
in lib/util/ModUtil.php at line 1272   + 
at ModUtil ::func ('Content', 'admin', 'editPage', array('pid' => '2')) 
in lib/Zikula/Bundle/CoreBundle/Controller/LegacyController.php at line 176   + 
at LegacyController ->getLegacyResponse ('Content', 'admin', 'editPage', array('pid' => '2'), false) 
in lib/Zikula/Bundle/CoreBundle/Controller/LegacyController.php at line 90   + 
at LegacyController ->legacyAction (object(Zikula_Request_Http)) 
at call_user_func_array (array(object(LegacyController), 'legacyAction'), array(object(Zikula_Request_Http))) 
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php at line 144   + 
at HttpKernel ->handleRaw (object(Zikula_Request_Http), '1') 
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php at line 64   + 
at HttpKernel ->handle (object(Zikula_Request_Http), '1', true) 
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php at line 69   + 
at ContainerAwareHttpKernel ->handle (object(Zikula_Request_Http), '1', true) 
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php at line 185   + 
at Kernel ->handle (object(Zikula_Request_Http)) 
in index.php at line 23   + 

```
### Steps to reproduce
- install ZK 1.4.3
- install Content master branch snapshot
- click "edit page" of sample content page

**Comment by Guite** [comment]
Fixed in #3094. Note you need to clear your `app/cache/` folder to let Symfony reload the updated service definitions.

**Comment by MrMontesa** [comment]
I can confirm that accessing the  "Content edit page" works now.
Unfortunately on saving the change, I receive a PageLock message popup that someone else is editing (my user name is displayed) and I should wait. This pupup is grayed out so I cannot click BACK, CHECK AGAIN or IGNORE LOCK}
Will open seperate ticket
